server/hack/deploy.sh

```diff
@@ -14,12 +14,14 @@ SERVER_LOG_LEVEL=${SERVER_LOG_LEVEL:-0}
 #
 # if the namespace doesn't exist, then head will return a failure (since it has
 # an empty input).  Since we manually check the error case, disable pipefail
-set +o pipefail
-toolchain_host=$(${KUBECLI} get namespaces -o name | grep toolchain-host | cut -d'/' -f2 | head -n 1)
-if [[ -z "${toolchain_host}" ]]; then
-    toolchain_host="toolchain-host_operator"
+if [[ -z "${TOOLCHAIN_HOST}" ]]; then
+    set +o pipefail
+    TOOLCHAIN_HOST=$(${KUBECLI} get namespaces -o name | grep toolchain-host | cut -d'/' -f2 | head -n 1)
+    if [[ -z "${TOOLCHAIN_HOST}" ]]; then
+        TOOLCHAIN_HOST="toolchain-host_operator"
```

Comment:
is this a typo?

```suggestion
        TOOLCHAIN_HOST="toolchain-host-operator"
```

Reply:
This was was was getting set before, and I didn't want to change it.

Why was it `toolchain-host_operator` to begin with?  Maybe this is something we need to change.

Reply:
I see  (https://github.com/konflux-workspaces/workspaces/blob/main/server%2Fhack%2Fdeploy.sh#L20). This seems a typo to me. I'd say we can fix it in this PR or in a separate one.

Reply:
Let's fix it in a separate PR, since it's unrelated to the work I'm looking to achieve in this PR.